Dockerfile

```diff
@@ -24,10 +25,10 @@ RUN LIBSODIUM_DOWNLOAD_URL="https://github.com/jedisct1/libsodium/releases/downl
     && cd libsodium-src \
     && ./configure && make -j$(nproc) && make install && ldconfig
 
-ENV REBAR3_VERSION="3.5.0"
+ENV REBAR3_VERSION="3.9.0"
```

Comment:
How does this affect 18.04, where according to [build](../../../aeternity/blob/master/docs/build.md) instructions the only thing that must be installed is Erlang from Ubuntu repositories?

Reply:
It does not affect [those build instructions](https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/docs/build.md), as those use `rebar3` binary in that repo:
* https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/.dockerignore#L5
* https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/Dockerfile#L4
* https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/Makefile#L4

Also, I see this image is [targeted specifically for CI](https://github.com/aeternity/docker-builder/blame/60305f9268463bb46edded55a3bed1d7d5a784a4/README.md#L3) - not users. I see rebar3 was added specifically for [one such repo's CI](https://github.com/aeternity/aesophia/pull/20).

Reply:
I saw rebar 3.5.0 embedded in `package_ubuntu_18.04/Dockerfile` - blocking until I understand that.

Reply:
I believe this shall be addressed by https://github.com/aeternity/docker-builder/pull/15